### ISSUE TITLE
Join effort with MathieuMoalic/vscode-mx3

### ISSUE THREAD

**Issue by MathieuMoalic** [JonathanMaes/mumax3-highlighter#1]
Hi there!
Some months ago, I published this vscode extension: https://github.com/MathieuMoalic/vscode-mx3 which is not used by a few people.
I wasn't able to make `mumax3-highlighter` work at all so it's probably still in a draft state.

I think `vscode-mx3`(mx3 Support) has the same aim as `mumax3-highlighter` so it might be more efficient if we joined effort on a single extension. 

What are your thoughts?

**Comment by JonathanMaes** [comment]
Hi, apologies for the late response.

I made this simple extension 3 years ago for myself, and earlier this year I just wondered how easy it is to publish extensions for VS Code and tested it on this one. It seems that your extension, on the other hand, was more than just an afterthought, so feel free to develop it further.
I recall searching for "mumax", "mumax3" and other such keywords in the VS Code extensions tab when I published this, but didn't find any extension except my own. Nowadays yours does show up when searching for those keywords, so that issue seems to be solved.
